plugins/GSdx/GSRendererOGL.cpp

```diff
@@ -606,6 +597,32 @@ void GSRendererOGL::SendDraw(bool require_barrier)
 		ASSERT(GLLoader::found_GL_ARB_texture_barrier);
 		glTextureBarrier();
 		dev->DrawIndexedPrimitive();
+	} else if (m_vt.m_primclass == GS_SPRITE_CLASS) {
+		size_t nb_vertex = (GLLoader::found_geometry_shader) ? 2 : 6;
+
+		GL_PUSH("Split the draw");
```

Comment:
Could you add something to seperate it from the other split draw code. For example  a `(SPRITE)` or `bounding box algo`.